File: matiasmasso/Mat.NET/DTO/DTOBasketCatalog.cs
using System;
using System.Collections.Generic;

namespace DTO
{

    public class DTOBasketCatalog
    {
        public List<Brand> brands { get; set; }

        public DTOBasketCatalog()
        {
            brands = new List<Brand>();
        }

        public class Brand
        {
            public Guid Guid { get; set; }
            public string nom { get; set; }
            public List<Category> categories { get; set; }

            public Brand(Guid oguid, string sNom)
            {
                Guid = oguid;
                nom = sNom;
                categories = new List<Category>();
            }
        }

        public class Category
        {
            public Guid Guid { get; set; }
            public string nom { get; set; }
            public List<Sku> skus { get; set; }

            public Category(Guid oguid, string sNom)
            {
                Guid = oguid;
                nom = sNom;
                skus = new List<Sku>();
            }
        }

        public class Sku
        {
            public Guid Guid { get; set; }
            public string nomCurt { get; set; }
            public string nomLlarg { get; set; }
            public decimal price { get; set; }
            public decimal dto { get; set; }
            public int moq { get; set; }
            public int stock { get; set; }
            public Sku(Guid oguid, string sNomCurt, string sNomLlarg)
            {
                Guid = oguid;
                nomCurt = sNomCurt;
                nomLlarg = sNomLlarg;
            }
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOBasketCatalog.vb
Public Class DTOBasketCatalog
    Property brands As List(Of Brand)

    Public Sub New()
        _brands = New List(Of Brand)
    End Sub

    Public Class Brand
        Property Guid As Guid
        Property nom As String
        Property categories As List(Of Category)

        Public Sub New(oguid As Guid, sNom As String)
            _guid = oguid
            _nom = sNom
            _categories = New List(Of Category)
        End Sub
    End Class

    Public Class Category
        Property Guid As Guid
        Property nom As String
        Property skus As List(Of Sku)

        Public Sub New(oguid As Guid, sNom As String)
            _guid = oguid
            _nom = sNom
            _skus = New List(Of Sku)
        End Sub
    End Class

    Public Class Sku
        Property Guid As Guid
        Property nomCurt As String
        Property nomLlarg As String
        Property price As Decimal
        Property dto As Decimal
        Property moq As Integer
        Property stock As Integer
        Public Sub New(oguid As Guid, sNomCurt As String, sNomLlarg As String)
            _guid = oguid
            _nomCurt = sNomCurt
            _nomLlarg = sNomLlarg
        End Sub
    End Class
End Class
